e2e/tshark/tshark_test.go

```diff
@@ -48,7 +49,7 @@ func TestTshark(t *testing.T) {
 	require.NoError(t, err, "error setting command")
 
 	t.Log("deploying minio as s3 backend")
-	err = kn.MinioClient.DeployMinio(ctx)
+	kn.MinioClient, err = minio.New(ctx, kn.K8sClient)
```

Comment:
you shouldn't need this anymore right? because you passed in the `MinioEnabled: true` option. 
This would overwrite the minio client that was already created. 

If overwriting was the goal, we should add a comment explaining why we are doing that.

Reply:
fixed